.github/workflows/smoke-test-daily.yml

```
@@ -18,21 +18,26 @@ jobs:
       - name: Checkout code.
         uses: actions/checkout@v2
         with:
-          path: package/woocommerce
+          ref: trunk
 
-      - name: Run npm install.
-        working-directory: package/woocommerce
-        run: npm install
-
-      - name: Move current directory to code. We will install zip file in this dir later.
-        run: mv ./package/woocommerce/* ./code/woocommerce
-
-      - name: Run login test.
-        working-directory: code/woocommerce
+      - name: Install prerequisites.
+        run: |
+          npm install
+          composer install --no-dev
+          npm run build:assets
+          npm install jest
+          
+     - name: Run smoke test.
```

Double check the spacing on this, I think this needs to be over to the right by one more space:

```suggestion
      - name: Run smoke test.
```